test/e2e/e2e_test.go

```diff
@@ -239,29 +241,7 @@ func namespaceRevision(tier v1alpha1.NSTemplateTier, typ string) (string, bool)
 	return "", false
 }
 
-func TestE2EFlowForMultipleAccounts(t *testing.T) {
```

Comment:
See https://github.com/codeready-toolchain/toolchain-e2e/commit/77c23fb4b10732473b141719e50b61cc4459d9fb#r36023806

Reply:
Incorporated multiple signup/mur creation/provisioning test in https://github.com/codeready-toolchain/toolchain-e2e/pull/49/commits/641334dfbc0edc8346af59e9ff8229a5ff5b9262

Reply:
Nice, I like the solution. Thanks for doing it :+1: